src/__tests__/__snapshots__/render.breaking.test.tsx.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`toJSON 1`] = `
```

Comment:
let's make the description a bit more descriptive, I don't know what this tests 😅

Reply:
LOL, this is duplicate of existing `render.test.tsx` :-)

Reply:
Then we can fix it in to places :D